src/components/widgets/MatchPreview.tsx

```diff
@@ -34,7 +34,7 @@ export default function MatchPreview(props: { match: MatchResponse; mainPlayerId
                 {player.mmr_diff! > 0 ? "↑" : "↓"}
                 {Math.abs(Math.round(player.mmr_diff!))}
               </span>
-              <Tooltip content={player.race}>
+              <Tooltip content={(player.race as string)[0].toUpperCase() + (player.race as string).slice(1)}>
```

Comment:
I think it looks better with an uppercase

Reply:
Can we move this to a central map from the enum to label instead?

Reply:
Sure, would be cleaner indeed.

Reply:
Done, did it for leagues as well